src/base/registrations/registrar.ts

```diff
@@ -109,34 +136,32 @@ export async function getAvatar(name: string, config: Config, resolver?: EnsReso
   return resolver.getText('avatar');
 }
 
-export async function getSeedHost(name: string, config: Config, resolver?: EnsResolver | null): Promise<string | null> {
+export async function getAnchorsAccount(name: string, config: Config, resolver?: EnsResolver | null): Promise<string | null> {
```

Comment:
In line with what we talked today, shouldn't we manage optional parameters only with `?` and pass only the required arguments to the functions after checking against `null`?

```suggestion
export async function getAnchorsAccount(name: string, config: Config, resolver?: EnsResolver): Promise<string | null> {
```

Reply:
We would need to create some temporary value for the `config.provider.getResolver(name)` call though, since we won't be able to associate it to `resolver`, ie.

```ts
// using var here to shadow the function param
const newResolver = resolver ?? await config.provider.getResolver(name);
if (! newResolver) {
  return null;
}
```

Reply:
Yeah, I wasn't going to do this in this PR, but let me check.

Reply:
Yeah I'd rather fix this separately.